src/components/Alert.ts

```diff
@@ -0,0 +1,14 @@
+import { SFC, createElement } from "react";
+import * as classNames from "classnames";
+
+export interface AlertProps {
+    message: string;
+    className?: string;
+}
+
+export const Alert: SFC<AlertProps> = ({ className, message }) =>
+    message
+        ? createElement("div", { className: classNames("alert alert-danger", className) }, message)
```

Comment:
Make generic... includes making sure your alert can support more than just the `danger` variety. Refer to other widgets, or Paul.